### DISCUSSION TITLE
temporary MQTT 0 values whithin reboot

### DISCUSSION THREAD

**Discussion by kalleick** [dewenni/ESP_Buderus_KM271#30] [Q&A] [answered]
i get temporary 0 values (e.g. ww_temp) over MQTT, if i do a reboot, is it possible to avoid this?

![2023-05-07 11_16_35-Home, Sweet Home](https://user-images.githubusercontent.com/95974901/236668802-22a9d062-639f-4901-8f2e-c92bf42431c1.png)

![2023-05-07 11_13_40-Home, Sweet Home](https://user-images.githubusercontent.com/95974901/236668792-ca664c5c-ce56-4947-9649-09ff44b35cbf.png)

**Comment by dewenni** [comment]
I must have a look on that, but I think that this is something that I can not avoid.
The values are send event driven and separated.
If the logamatic send a value it is directly send by MQTT.
So normally the values should be valid if they are send, because they are only send if there is an incoming message from the logamatic.

Let me see if I can also reproduce this with my setup.
I will come back on this later.

**Reply by dewenni** [reply, marked as answer]
Ok, I think I got it.

There is a additional MQTT message (esp_heizung/info) that contain a payload in json format.

{
  "burner": 0,
  "pump": 0,
  "ww_temp": 51,
  "boiler_temp": 43
}

That was just a little gimmick. This message contains 4 interesting values and is send in a cyclic period.

If the esp restarted, the message could be send before the logamatic has send all the 4 containing values.
Maybe I should delay this message until all Status values are received at least one time.

I would suggest to use the "normal" status values.
Here you have also the same 4 values available.
